src/hotspot/share/classfile/classLoaderData.hpp

```diff
@@ -124,7 +124,6 @@ class ClassLoaderData : public CHeapObj<mtClass> {
 
   // Remembered sets support for the oops in the class loader data.
   bool _modified_oops;             // Card Table Equivalent (YC/CMS support)
-  bool _accumulated_modified_oops; // Mod Union Equivalent (CMS support)
```

Comment:
Maybe remove the 'CMS support' comment on line 126?

Reply:
Ok, can remove that too.

Reply:
Oh should I take the entire comment out?  I can do that. YC isn't helpful.